codec_specs.md

```diff
@@ -815,24 +819,30 @@ Codec Name: UTF-8 Plain Text
 
 Description: Basic text subtitles. For more information, see (#subtitles) on Subtitles.
 
+Initialization: none
+
 ### S_TEXT/SSA
 
 Codec ID: S_TEXT/SSA
 
 Codec Name: Subtitles Format
 
-Description: The [Script Info] and [V4 Styles] sections are stored in the codecprivate. Each event is stored in its own Block.
+Description: Each event is stored in its own Block.
 For more information, see (#ssa-ass-subtitles) on SSA/ASS.
 
+Initialization: The `Private Data` contains the [Script Info] and [V4 Styles] sections.
+
 ### S_TEXT/ASS
 
 Codec ID: S_TEXT/ASS
 
 Codec Name: Advanced Subtitles Format
 
-Description: The [Script Info] and [V4 Styles] sections are stored in the codecprivate. Each event is stored in its own Block.
+Description: Each event is stored in its own Block.
 For more information, see (#ssa-ass-subtitles) on SSA/ASS.
 
+Initialization: The `Private Data` contains the [Script Info] and [V4 Styles] sections.
```

Comment:
Unfortunately, the original text was wrong. For `S_TEXT/ASS` the two sections are `[Script Info]` (this one was correct) and `[V4+ Styles]` (this one was incorrect; it's `[V4 Styles]` without the `+` for SSA but `[V4+ Styles]` with the `+` for ASS). So let's correct it while we touch this part anyway.

Thanks!